### DISCUSSION TITLE
C2 IP'S into Intel SO 2.3.61

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#4982] [Unsupported Versions] [answered]
Hello Everyone! 

Question? Is it possible to load known C2 server IP's and Domain names into the Intel functionality for Sec Onion. For example, If DFIR published daily C2 IP's something like the link below, would I be able to add the IP's to the intel config for alerting?

Anyone have experience doing this?

https://thedfirreport.com/services/
https://docs.zeek.org/en/master/frameworks/intel.html

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/zeek.html#intel
